[BUGFIX] Rel attr setting

scribblecrumb opened the pull request:
<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
<!-- Be sure to name your PR something descriptive and succinct; include Bugfix: Feature: Enhancement: or Content: in the title to describe what type of PR it is. -->
<!-- IF YOU ARE DOING A BUGFIX: Please target the latest release branch if the bug that you are fixing is also present in the latest release. -->

## About The Pull Request
Mostly just more instances of improperly settng rel attrs.  Also realized that we weren't *actually* adding to romance during romance events, nor were we applying rel increases when the intensity was anything other than high.... ugh
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage senior developers from merging your PR! -->

## Proof of Testing
Using debug mode to display what's happening:
<img width="299" height="111" alt="image" src="https://github.com/user-attachments/assets/3f229410-acf9-46a2-a93b-356edb8fd11a" />

Romance before interaction

<img width="297" height="119" alt="image" src="https://github.com/user-attachments/assets/6aa20c1f-fa63-425b-b9c5-5e08adebf75d" />

Romance after the interaction

<img width="431" height="159" alt="image" src="https://github.com/user-attachments/assets/438eaf17-9f4a-4255-9082-4388dd853e4c" />

This is an interaction where respect is increased, here's respect before the increase

<img width="285" height="147" alt="image" src="https://github.com/user-attachments/assets/9af5ee54-df7e-4e3c-9624-ef0419ead534" />

And respect after.

<!-- Include any screenshots, debugging steps, or links to beta testing threads here. At least one form of proof of testing is REQUIRED for all new content. You must be able to run the code locally before you PR it here. -->

## Changelog/Credits
- Relationship events now properly apply relationship changes
<!-- Include any changes that should be made to the changelog of the game here or any changes to the credits file of the game. -->
<!-- This is just for easy access later for senior developers gathering this information; this process is not automated. -->

scribblecrumb committed:
fix attr setting

larkgz commented on scripts/cat_relations/relationship.py:
```diff
@@ -308,14 +308,24 @@ def interaction_affect_relationships(
+                    rel_out,
+                    getattr(self, rel_out) + choice(buffs)
+                    if rel_type != rel_out
+                    else amount,
```
It seems like this can still set the relationship to exactly `amount` (which seems to be how much the number is supposed to change by), is this intended?

scribblecrumb replied:
Ah I think I was assuming the if else was isolated to `choice(buffs)` and didn't include the `getattr()`.  I've added parentheses now so that it works as intended.

scribblecrumb committed:
Update relationship.py

- remove unused buffs = [] assignment
- add parentheses around `buff if else amount` statement
